packages/core/src/components/forms/_controls.scss

```diff
@@ -674,4 +674,129 @@ $control-checked-background-color-active: nth(map-get($button-intents, "primary"
 
   Styleguide components.forms.checkbox.inline
   */
+
+
+  /*
+  Numeric steppers
+
+  Numeric steppers provide controls for easily incrementing or decrementing a numeric value.
+
+  ##### Keyboard interactions
+
+  - `↑` - increment the value by `stepSize` (default: `1`)
+  - `↓` - decrement the value by `stepSize` (default: `1`)
+  - `Shift + ↑` - increment the value by `majorStepSize` (default: `10`)
+  - `Shift + ↓` - decrement the value by `majorStepSize` (default: `10`)
+  - `Alt/Option + ↑` - increment the value by `minorStepSize` (default: `0.1`)
+  - `Alt/Option + ↓` - decrement the value by `minorStepSize` (default: `0.1`)
+  - `Enter` - triggers the `onDone` callback if one was provided; otherwise, sets invalid input to `0`
+
+  ##### Mouse interactions
+
+  - `Click '+'` - increment the value by `stepSize` (default: `1`)
+  - `Click '-'` - decrement the value by `stepSize` (default: `1`)
+  - `Shift + Click '+'` - increment the value by `majorStepSize` (default: `10`)
+  - `Shift + Click '-'` - decrement the value by `majorStepSize` (default: `10`)
+  - `Alt/Option + Click '+'` - increment the value by `minorStepSize` (default: `0.1`)
+  - `Alt/Option + Click '-'` - decrement the value by `minorStepSize` (default: `0.1`)
+
+  ##### Example
+
+  The following example is a configurable `NumericStepper` that responds to the keyboard and mouse interactions listed above.
+
+  @react-example NumericStepperExample
+
+  Weight: 10
+
+  Styleguide components.forms.numeric-stepper
+  */
+
+  /*
+  Javascript API
+
+  The `NumericStepper` component is available in the __@blueprintjs/core__ package.
+  Make sure to review the [general usage docs for JS components](#components.usage).
+
+  A numeric stepper can function as either an uncontrolled component or a semi-controlled component. It cannot
+  function as a purely controlled component, because it needs to adjust its own copy of the `value` internally
+  in response to keyboard and mouse interactions.
+
+  ##### Uncontrolled mode
```

Comment:
I don't understand the subsection/"weight" stuff, so it's way trickier for me to change to the `/* ... */` syntax successfully. Could use a hand here.